### PR TITLE
fix: Update instructions following the Airbyte Cloud integration

### PR DESCRIPTION
**Pull request by cgkournelos-rf**
Minor changes that improve the understanding of the content.
These changes are based on an internal review process.

**UPDATE** this PR to match the cloud integration.

Linear ticket [TECH-741](https://linear.app/kyve/issue/TECH-741/update-docs-following-the-airbyte-cloud-integration)

Main changes:
- Add a reference to Airbyte Cloud
- Change the repo link for cloning to the official airbyte instead of the KYVE fork 
- Remove the building process of `source-kyve` since it is already integrated

### CODE REVIEW

File: docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md [hunk 1]
```diff
@@ -74,6 +74,8 @@ Now you're ready to create the connection in Airbyte (<http://localhost:8000/>).
 
 1. Set up the source.
 
+   Search "KYVE" in the `Sources searchbar` and select it.
```
Comment (eferm) [inline docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md]:
```suggestion
   Search for "KYVE" in the `Sources search bar` and select it. 
```

File: docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md [hunk 2]
```diff
@@ -88,8 +90,13 @@ Now you're ready to create the connection in Airbyte (<http://localhost:8000/>).
    <br></br>
    <img src="/img/elt/airbyte_kyve_source.jpg" alt="Airbyte set up source" />
 
+   ***NOTE***: The **Bundle-Start-IDs** determines the size of the extracted data. This also affects the time that it is required for completing
+   the sync process which can take place for hours if the data are old. It is advised for testing purposes to visit the [KYVE network Polls page](https://app.korellia.kyve.network/#/pools) and to select a recent bundle ID (2-5 days before).
```
Comment (eferm) [inline docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md]:
```suggestion
   ***NOTE***: The **Bundle-Start-IDs** determines the size of the extracted data. This also affects the time required for completing
   the sync process, up to several hours for historic data. It is advised for testing purposes to visit the [KYVE network Polls page](https://app.korellia.kyve.network/#/pools) and to select a recent bundle ID (2-5 days old).
```

File: docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md [hunk 3]
```diff
@@ -88,8 +90,13 @@ Now you're ready to create the connection in Airbyte (<http://localhost:8000/>).
...
+
 2. Set up the destination.
 
+   Search "Postgres" in the `Destinations searchbar` and select it.
```
Comment (eferm) [inline docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md]:
```suggestion
   Search for "Postgres" in the `Destinations search bar` and select it.
```

File: docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md [hunk 4]
```diff
@@ -102,10 +109,11 @@ Now you're ready to create the connection in Airbyte (<http://localhost:8000/>).
 
 3. Final ELT configuration.
 
-   This is the final step where you can modify the pipeline.
+   Select the `KYVE Source` that you have configured in step 1 and the `Postgres Destination` from step 2.
+   In this final step  you can modify the pipeline.
```
Comment (eferm) [inline docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md]:
```suggestion
   In this step you can make final modifications to the pipeline.
```

**Review by eferm — DISMISSED** [review]
lg! I added some optional suggestions around clarity; feel free to "merge suggestion" if useful.

**Comment by cgkournelos-rf** [comment]
@eferm Thanks for your suggestions, I just added them.

Do you know why the reviewdog is failing?

**Comment by eferm** [comment]
@cgkournelos-rf I think it's always failing for any PR... I believe we've just ignored it? we should definitely fix it.

**Comment by harrytou** [comment]
This PR is linked to https://linear.app/kyve/issue/TECH-741/update-docs-following-the-airbyte-cloud-integration

**Comment by cgkournelos-rf** [comment]
@eferm This PR is updated to follow Linear ticket [TECH-741](https://linear.app/kyve/issue/TECH-741/update-docs-following-the-airbyte-cloud-integration)

File: docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md [hunk 5]
```diff
@@ -88,8 +90,13 @@ Now you're ready to create the connection in Airbyte (<http://localhost:8000/>).
    <br></br>
    <img src="/img/elt/airbyte_kyve_source.jpg" alt="Airbyte set up source" />
 
+   ***NOTE***: The **Bundle-Start-IDs** determines the size of the extracted data. This also affects the time required for completing
+   the sync process, up to several hours for historic data. It is advised for testing purposes to visit the [KYVE network Polls page](https://app.korellia.kyve.network/#/pools) and to select a recent bundle ID (2-5 days old).
+
 2. Set up the destination.
 
+   Search for "Postgres" in the `Destinations search bar` and select it.
```
Comment (harrytou) [inline docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md]: @cgkournelos-rf Convert it to "`Destinations` search bar".

File: docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md [hunk 6]
```diff
@@ -74,6 +74,8 @@ Now you're ready to create the connection in Airbyte (<http://localhost:8000/>).
 
 1. Set up the source.
 
+   Search for "KYVE" in the `Sources search bar` and select it.
```
Comment (harrytou) [inline docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md]: @cgkournelos-rf Convert it to "`Sources` search bar".

File: docs/data_engineers/accessing_data/elt_pipeline/overview.md [hunk 7]
```diff
@@ -9,60 +9,14 @@ supported by [Airbyte](https://airbyte.com/). With the [ELT](https://en.wikipedi
...
 
-To experiment with the ELT pipeline via [Airbyte](https://airbyte.com/) let's first ensure your computer is configured correctly.
+For users that wants to experiment locally with the KYVE data it is recommended the Local Deployment approach of Airbyte. This option requires some configuration steps in order to run locally the Airbyte app. Detailed instructions are presented in [Airbyte Local Deployment](airbyte_local_deployment) section.
 
-### Windows
+## Airbyte Cloud
 
-For Windows users, the Windows Subsystem for Linux 2 (WSL2) must be installed. You must be running Windows 10 version
-2004 or higher (build 19041 or higher), or Windows 11 to use the commands in this tutorial.
+You can connect to the [Airbyte Cloud](https://airbyte.com/airbyte-cloud) platform and try the [KYVE Source Connector](https://cloud.airbyte.com/workspaces/2a2bfca3-c289-48af-a11d-1a9ad69ea187/source/new-source/60a1efcc-c31c-4c63-b508-5b48b6a9f4a6).
```
Comment (eferm) [inline docs/data_engineers/accessing_data/elt_pipeline/overview.md]: Is this workspace publicly available? The link doesn't work for me.

File: docs/data_engineers/accessing_data/elt_pipeline/overview.md [hunk 8]
```diff
@@ -9,60 +9,14 @@ supported by [Airbyte](https://airbyte.com/). With the [ELT](https://en.wikipedi
...
-
-   ```sh
-   brew install --cask docker
-   ```
-
-### Linux
-
-1. In order to install Docker and `docker-compose` on Linux, please follow the instructions
-   listed [here](https://docs.docker.com/engine/install/) and choose your Linux distribution. Remember to also run the post-installation steps listed [here](https://docs.docker.com/engine/install/linux-postinstall/).
+This option does not require any local setup. For setting up destination you could follow the documentation inside the Airbyte cloud page. [Snowflake](elt_destinations/snowflake) section presents a destination example that could be used with cloud.
```
Comment (eferm) [inline docs/data_engineers/accessing_data/elt_pipeline/overview.md]:
```suggestion
This option does not require any local setup. For setting up a destination follow the documentation in the Airbyte cloud page. For example, the [Snowflake](elt_destinations/snowflake) section presents one such destination that could be used with cloud.
```

File: docs/data_engineers/accessing_data/elt_pipeline/airbyte_local_deployment.md [hunk 9]
```diff
@@ -4,12 +4,70 @@ sidebar_position: 1
...
+
+## Local Deployment
+
 In this step, we'll deploy Airbyte locally on your machine. Make sure you have installed Docker and Docker Compose on your system as shown in the [Overview section](overview.md).
 
 To deploy Airbyte locally, run the following commands in your terminal:
 
 ```sh
-git clone https://github.com/KYVENetwork/airbyte.git
+git clone https://github.com/airbytehq/airbyte.git
```
Comment (eferm) [inline docs/data_engineers/accessing_data/elt_pipeline/airbyte_local_deployment.md:70]: 🎉!

**Review by eferm — DISMISSED** [review]
Approved but please review the Airbyte Workspaces comment before merging.

File: docs/data_engineers/accessing_data/elt_pipeline/overview.md (hunk 7 above)
Comment (cgkournelos-rf) [reply docs/data_engineers/accessing_data/elt_pipeline/overview.md]: You were right this link is not working for every user, so we will not use a direct link.

**Review by harrytou — DISMISSED** [review]

**Review by harrytou — DISMISSED** [review]

**Review by harrytou — DISMISSED** [review]

**Review by harrytou — APPROVED** [review]

**Pull request merged**